region/maridia/outer/Mama Turtle Room.json

```diff
@@ -760,6 +765,18 @@
       "name": "Base",
       "requires": []
     },
+    {
+      "id": 32,
+      "link": [4, 6],
+      "name": "Base",
+      "requires": [
+        {"or": [
+          "canWalljump",
+          "SpaceJump",
+          "HiJump"
```

Comment:
okay, 4->6 with HiJump is not bad, and 4->3 without HiJump, shooting out the block, is a separate strat still